tools/gen_usb_descriptor.py

```diff
@@ -131,7 +132,7 @@ def strings_in_order(cls):
                 bInterval=0),
             standard.EndpointDescriptor(
                 description="MSC out",
-                bEndpointAddress=0x1 | standard.EndpointDescriptor.DIRECTION_OUT,
+                bEndpointAddress=0x0 | standard.EndpointDescriptor.DIRECTION_OUT,
```

Comment:
Can you test this on a SAMD21 and SAMD51? I feel like I tried it and it didn't work.

Reply:
Testing today

Reply:
SAMD51 is fine but the 21 has write issues. Reverting until Dan's PR is in.